scope3_methodology/test/defaults/atp-defaults.yaml

```diff
@@ -0,0 +1,265 @@
+defaults:
```

Comment:
how is this generated/maintained? and then how is this used in testing?

Reply:
Right now, I generated the default files and copied them into this directory for testing use. These files are directly used in both testing files. This may change in the future, when I update the testing for computing default files https://github.com/scope3data/methodology/issues/52 But for now, so we have testing these are manually updated here.